Marlin/src/pins/pins_RAMPS_X86.h

```diff
@@ -0,0 +1,572 @@
+/**
```

Comment:
This can just be `#include "pins_RAMPS.h"` if there are no differences between them.

Reply:
I was keeping them separate in case it needed modification, wasn't sure where things were heading when I started.